ci/new-instance.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -eu
+
+function log {
+   echo "$(date +"%T") - INFO - $*"
+}
+
+read -r -p "Cluster (test or production): " CLUSTER
+
+if [ -z "${CLUSTER}" ]; then
+    log "Nothing typed"
+    exit 2
+fi
+
+log "CLUSTER: $CLUSTER"
+
+read -r -p "lumen url:" URL
+
+if [ -z "${URL}" ]; then
+    log "Nothing typed"
+    exit 2
+fi
+
+read -r -p "lumen organisation title:" ORGANISATION
+
+if [ -z "${ORGANISATION}" ]; then
+    log "Nothing typed"
+    exit 2
+fi
+
+read -r -p "user email:" EMAIL
+
+if [ -z "${EMAIL}" ]; then
+    log "Nothing typed"
+    exit 2
+fi
+
+
+log "URL: $URL"
+log "ORGANISATION $ORGANISATION"
+log "EMAIL $EMAIL"
+
+gcloud container clusters get-credentials "$CLUSTER" --zone europe-west1-d --project akvo-lumen
+
+export POD_NAME=$(kubectl get pods -l "run=lumen-blue-green,color=$(./live-color.sh)" -o jsonpath="{.items[0].metadata.name}")
```

Comment:
The "./live-color.sh" path means that the expectation is that the script is run from within the `ci` directory. 

Either:
1. Add this to the documentation.
2. Check that the POD_NAME is not empty
3. Avoid the issue as per https://github.com/akvo/akvo-lumen/blob/4b1ac11ac88c8869fa189f8fcd141e1b84843333/ci/flip-blue-green-deployment.sh#L35

Reply:
included